tests/features/solution/add_solution.feature

```diff
@@ -35,7 +38,11 @@ Feature: "Add solution" visibility options.
     When I fill in the following:
       | Title             | Collection solution add solution |
       | Description       | This is a test text              |
-      | Documentation     | text.pdf                         |
+    And I attach the file "text.pdf" to "Documentation"
```

Comment:
The previous way never worked. The file won't be uploaded at all, because in symfony/dom-crawler/Field/FileFormField.php:55 a call to `is_readable()` is done, but text.pdf is not readable at all. The "attach the file" step will actually convert the file name to a proper full path.